dockerfile

```
@@ -21,8 +21,11 @@ RUN cp /etc/nuoj/setting/setting.json /etc/nuoj/setting.json
 
 # install tailwindcss and react.js from npm
 WORKDIR /etc/nuoj
-RUN npm install -D tailwindcss
-RUN npm install babel-cli@6 babel-preset-react-app@3
+# RUN npm install -D tailwindcss
+# RUN npm install babel-cli@6 babel-preset-react-app@3
+RUN npm install
```

可以考慮用 [PnPm](https://pnpm.io/)

```suggestion
RUN npm install pnpm
```